### PR TITLE
[docs] Reference Elastic Integrations Contribution docs

### PR DESCRIPTION
**Pull request by nicpenning**
This adds the link to the contribution docs for Elastic Integrations.

image

Closes: #1974

### CODE REVIEW

File: tools/readme/readme.md.tmpl [hunk 1]
```diff
@@ -9,6 +9,9 @@ test and build your packages. Learn about each of these and other features in [_
 
 Currently, `elastic-package` only supports packages of type [Elastic Integrations](https://github.com/elastic/integrations).
 
+Please review the [Contributing Guide](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md) to learn how to build and develop packages, understand the release procedure and
```
Comment (jsoriano) [inline tools/readme/readme.md.tmpl:12]: Nit. Maybe we should be more specific about this contributing guide referring to packages/integrations and not about contributing to `elastic-package` itself.
```suggestion
Please review the [integrations contributing cuide](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md) to learn how to build and develop packages, understand the release procedure and
```

**Review by jsoriano — APPROVED** [review]

**Comment by nicpenning** [comment]
Good suggestion, I applied it.

**Review by jsoriano — APPROVED** [review]

**Comment by nicpenning** [comment]
Will this build and update the readme?

**Comment by nicpenning** [comment]
Got both Readmes updated and there was a typo I corrected. A re-test will be needed.

**Comment by nicpenning** [comment]
Excellent, thanks!

**Pull request merged**